pycloudflare/models.py

```diff
@@ -113,7 +113,8 @@ def records(self):
         return by_name
 
     def create_record(self, name, type, content, ttl=1, proxied=False,
-                      priority=10):
+                      priority=10, service=None, protocol=None, weight=0,
+                      port=None, target=None):
```

Comment:
Having keywords has it's own positives, but I think it might look cleaner to use `**kwargs` for arguments that aren't gonna exist for every record type.